README.md

```diff
@@ -3,11 +3,11 @@
 **Donate Link:** http://tareq.co/donate/  
 **Tags:** WooCommerce multivendor marketplace, multi vendor marketplace, multi seller store, multi-vendor, multi seller, commissions, multivendor, marketplace, product vendors, woocommerce vendor, commission rate, e-commerce, woocommerce, ebay, ecommerce.  
 **Requires at least:** 5.4  
-**Tested up to:** 6.2.0  
+**Tested up to:** 6.2.2  
```

Comment:
kindly create a new branch with the latest version of the develop branch and add your changes there.

Reply:
What I did is, rebased this branch with latest develop branch code. Apparently it has the same effect.